src/clients/apiClient.ts

```diff
@@ -145,6 +145,17 @@ export async function fetchLogin(encodedUsernamePassword: string): Promise<void>
 	}
 }
 
+export async function fetchIsAdmin(encodedUsernamePassword: string): Promise<void> {
+	const response = await fetch(`${backendUrl}/login/admin`, {
```

Comment:
We have an endpoint called `getUserById` in the user controller, we can call this and then return the `UserType` as a boolean